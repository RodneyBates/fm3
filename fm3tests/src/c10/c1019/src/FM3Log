[32mFM3:   [0mCompiling with command line: 
         /home/rodney/proj/m3/fm3/git/compiler/AMD64_LINUX/FM3 --disasm --no-disasm-verbose --exprs --no-expr-addrs --no-std-sources Main.m3"
[32mFM3:   [0mGetting dependencies of 
         /home/rodney/proj/m3/fm3/git/fm3tests/src/c10/c1019/src/Main.m3 ...
[32mFM3:   [0mGetting dependencies of 
         /home/rodney/proj/m3/fm3/git/fm3tests/src/c10/c1019/src/I1.i3 ...
[32mFM3:   [0mCompiling I1.i3 ...
[32mFM3:   [0mFinished compiling I1.i3.
[32mFM3:   [0mCompiling Main.m3 ...
[32mFM3:   [0mFinished compiling Main.m3.
