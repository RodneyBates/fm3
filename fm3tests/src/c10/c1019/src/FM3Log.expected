Start unit Main.m3
[32mFM3: [0mCompiling /home/rodney/proj/m3/fm3/git/fm3tests/src/c10/c1019/src/Main.m3...
Start unit Main.i3
[32mFM3: [0mCompiling /home/rodney/proj/m3/fm3/git/fm3tests/src/c10/c1019/src/Main.i3...
[32mFM3: [0mPass 1 output file Main.i3.FM3Pass1 has 133 bytes.
[32mFM3: [0mPatch stack Main.i3.FM3Patch peak size = 39 bytes.
[32mFM3: [0mPass 2 output file Main.i3.FM3Pass2 has 106 bytes.
[32mFM3: [0mFinished compiling Main.i3.
End unit Main.i3
Start unit I1.i3
[32mFM3: [0mCompiling /home/rodney/proj/m3/fm3/git/fm3tests/src/c10/c1019/src/I1.i3...
[32mFM3: [0mPass 1 output file I1.i3.FM3Pass1 has 75 bytes.
[32mFM3: [0mPatch stack I1.i3.FM3Patch peak size = 32 bytes.
[32mFM3: [0mPass 2 output file I1.i3.FM3Pass2 has 57 bytes.
[32mFM3: [0mFinished compiling I1.i3.
End unit I1.i3
[32mFM3: [0mPass 1 output file Main.m3.FM3Pass1 has 659 bytes.
[32mFM3: [0mPatch stack Main.m3.FM3Patch peak size = 52 bytes.
[32mFM3: [0mPass 2 output file Main.m3.FM3Pass2 has 514 bytes.
[32mFM3: [0mFinished compiling Main.m3.
End unit Main.m3
